### ISSUE TITLE
[Bug] Yggdrasil process eating lots of CPU and slowing down network

### ISSUE THREAD

**Issue by lemonzest79** [yggdrasil-network/yggdrasil-go#1167]
Hi

I'm on Fedora 40 Cinnamon and install yggdrasil from the copr repo, since the last update I've noticed the yggdrasil process eating a lot of CPU (I have btop open constantly) and it seemed to be affecting networking, my wireless devices were running really slow but when I disabled the yggdrasil service `sudo systemctl disable --now yggdrasil.service` on all machines full network speed resumed, I've no idea what is going on with this, I have it installed an approx 10 machines at home, mostly fedora 40 but 4 debian sid machines too

**Comment by neilalexander** [comment]
How many peers? If you do `yggdrasilctl getPeers` do you see large amounts of TX/RX traffic?

**Comment by lemonzest79** [comment]
Yes there was lots of traffic, the 10 local machines here each had an in/out and then there was the public peer

**Comment by neilalexander** [comment]
You'll need to be more specific about which peers are the ones with high traffic movement, as it is possible that your computer was just being used to route a lot of traffic (far more likely with a higher number of peers).

**Comment by lemonzest79** [comment]
I think I know whats causing the wireless slowdown its the wireless machines all talking to each other through the wifi and like 5 devices is slowing the wifi down, heres the results from one machine thats running high cpu %


![yggdrasil0](https://github.com/user-attachments/assets/5f51586f-ef15-4a88-9ad5-2f39114ab513)
![yggdrasil1](https://github.com/user-attachments/assets/c3734af5-9752-48aa-8b06-c64aa231af1e)

**Comment by Vort** [comment]
> its the wireless machines all talking to each other through the wifi

State ... Down ... 0s ago looks suspicious.
Are they constantly connecting and disconnecting?
